omod/src/main/java/org/openmrs/module/registrationapp/RegistrationAppUiUtils.java

```diff
@@ -180,14 +181,13 @@ public static void checkForIdentifierExceptions(Exception ex, Errors errors) {
 		if (ex instanceof PatientIdentifierException) {
 			if (ex instanceof InvalidCheckDigitException) {
 				errors.reject("registrationapp.error.identifier.invalidCheckDigit");
-			}
-			else if (ex instanceof DuplicateIdentifierException) {
+			} else if (ex instanceof DuplicateIdentifierException) {
```

Comment:
@areklalo  thanks for working on this lalo,got to see some simple code formats in this!!